boards/risc-v/esp32c6/common/scripts/esp32c6_sections.ld

```diff
@@ -440,37 +440,105 @@ SECTIONS
     . = ALIGN (4);
   } > default_rodata_seg AT > ROM
 
-  /* RTC fast memory holds RTC wake stub code !*/
-
+  /**
+   * RTC fast memory holds RTC wake stub code,
+   * including from any source file named rtc_wake_stub*.c
+   */
   .rtc.text :
   {
     . = ALIGN(4);
-    *(.rtc.literal .rtc.text)
+    _rtc_fast_start = ABSOLUTE(.);
+    _rtc_text_start = ABSOLUTE(.);
     *(.rtc.entry.text)
+
+    mapping[rtc_text]
+
+    *rtc_wake_stub*.*(.literal .text .literal.* .text.*)
+    *(.rtc_text_end_test)
+
     /* 16B padding for possible CPU prefetch and 4B alignment for PMS split lines */
-    . += 16;
+    . += 0;
     . = ALIGN(4);
 
-  } > lp_ram_seg AT > ROM
-
-  /* RTC BSS section. */
+    _rtc_text_end = ABSOLUTE(.);
+  } > lp_ram_seg
 
-  .rtc.bss (NOLOAD) :
+  /**
+   * This section located in RTC FAST Memory area.
+   * It holds data marked with RTC_FAST_ATTR attribute.
+   * See the file "esp_attr.h" for more information.
+   */
+  .rtc.force_fast :
   {
-    *(.rtc.bss)
+    . = ALIGN(4);
+    _rtc_force_fast_start = ABSOLUTE(.);
+
+    mapping[rtc_force_fast]
+
+    *(.rtc.force_fast .rtc.force_fast.*)
+    . = ALIGN(4) ;
+    _rtc_force_fast_end = ABSOLUTE(.);
   } > lp_ram_seg
 
+  /**
+   * RTC data section holds RTC wake stub
+   * data/rodata, including from any source file
+   * named rtc_wake_stub*.c and the data marked with
+   * RTC_DATA_ATTR, RTC_RODATA_ATTR attributes.
+   */
   .rtc.data :
   {
     _rtc_data_start = ABSOLUTE(.);
-    *(.rtc.data)
-    *(.rtc.data.*)
-    *(.rtc.rodata)
-    *(.rtc.rodata.*)
+
+    mapping[rtc_data]
```

Comment:
This syntax is not valid in NuttX (this is used on ESP-IDF, as you can check [here](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-guides/linker-script-generation.html)). Can you double check this syntax and revert to the old (default for linker script) one?